packages/core/core.v0.12.1/opam

```diff
@@ -10,7 +10,7 @@ build: [
   ["dune" "build" "-p" name "-j" jobs]
 ]
 depends: [
-  "ocaml"        {>= "4.07.0"}
+  "ocaml"        {>= "4.07.0" & "4.08.0"}
```

Comment:
```suggestion
  "ocaml"        {>= "4.07.0" & < "4.08.0"}
```
did you want to say < 4.08? (same for the other files)

Reply:
Sure; thanks!

Reply:
could you do the same for the other packages?

Reply:
Sorry, I misinterpreted your message;
https://github.com/ocaml/opam-repository/pull/14212/commits/5cac02cb7f008223397f623fb39f897d03b73079 should fix the constraints.